client/src/pages/events/Show.js

```diff
@@ -0,0 +1,288 @@
+import { DEFAULT_PAGE_PROPS, DEFAULT_SEARCH_PROPS } from "actions"
+import API from "api"
+import AppContext from "components/AppContext"
+import DictionaryField from "components/DictionaryField"
+import * as FieldHelper from "components/FieldHelper"
+import Fieldset from "components/Fieldset"
+import LinkTo from "components/LinkTo"
+import Messages from "components/Messages"
+import NoPaginationOrganizationTable from "components/NoPaginationOrganizationTable"
+import NoPaginationPersonTable from "components/NoPaginationPersonTable"
+import NoPaginationTaskTable from "components/NoPaginationTaskTable"
+import {
+  jumpToTop,
+  mapPageDispatchersToProps,
+  PageDispatchersPropType,
+  SubscriptionIcon,
+  useBoilerplate,
+  usePageTitle
+} from "components/Page"
+import ReportCollection from "components/ReportCollection"
+import RichTextEditor from "components/RichTextEditor"
+import { Field, Form, Formik } from "formik"
+import { Event, Report, Task } from "models"
+import moment from "moment/moment"
+import pluralize from "pluralize"
+import React, { useContext, useState } from "react"
+import { connect } from "react-redux"
+import { useLocation, useParams } from "react-router-dom"
+import Settings from "settings"
+import { FORMAT_TABLE } from "../../components/ReportCollection"
```

Comment:
Please import directly from "components/ReportCollection" (joined with the import several lines above).